src/Sulu/Bundle/AdminBundle/Resources/js/components/FileUploadButton/FileUploadButton.js

```diff
@@ -39,7 +41,7 @@ export default class FileUploadButton extends React.Component<Props> {
                         <Button disabled={disabled} icon={icon} skin={skin}>
                             {children}
                         </Button>
-                        <input {...getInputProps()} />
+                        <input {...getInputProps()} accept={accept} />
```

Comment:
This should  handled over the dropzone params so if you configure `accept` on `<Dropzone` the `getInputProps` will return it correctly.

If we add the `accept` here, we need to make sure this is also possible for all other dropzone components: Quick search:

 - FileUploadButton
 - UploadToolbarAction
 - SingleMediaDropzone
 - MultiMediaDropzone

The integration into sulu should then need to support to configure this new accept prop, as example the `sulu_admin.upload` toolbar action should be possible to be configured with the accept and the `single_media_upload` field type.

/cc @niklasnatter do you see other places where the accept should be kept in mind?

Reply:
@alexander-schranz Yes indeed, i believe the `Dropzone` from `UploadToolbarAction` already has an `accept` property.

Reply:
You are right 👍 still would add accept to SingleMediaDropzone and MultiMediaDropzone also.

Reply:
I have added accept for SingleMediaDropzone and MultiMediaDropzone